src/main/resources/templates/email/generic/body.ftlh.mjml

```diff
@@ -0,0 +1,17 @@
+<!-- <#-- @ftlvariable name="" type="com.terraformation.backend.email.model.GenericEmail" --> -->
+<mjml>
+    <mj-include path="../head.ftlh.mjml"/>
+
+    <mj-body>
+        <mj-include path="../logo.ftlh.mjml"/>
+
+        <mj-section padding="0px">
+            <mj-column mj-class="body-wrapper">
+                <mj-text mj-class="text-body03" padding-top="24px" padding-bottom="24px">
+                ${emailBody?no_esc}
+                </mj-text>
+                <mj-include path="../footer.ftlh.mjml"/>
+            </mj-column>
+        </mj-section>
+    </mj-body>
+</mjml>
```

Comment:
Trailing newlines would be preferable on these files just to keep git from flagging the lack of them.

Reply:
That's fine with me. I must not have my auto-save formatter setup to add those for me